webui/react/src/pages/ExperimentDetails/ExperimentTrials.tsx

```diff
@@ -101,6 +101,13 @@ const ExperimentTrials: React.FC<Props> = ({ experiment }: Props) => {
       </Link>
     );
 
+    const restartsRenderer = (_: string, record: TrialItem): React.ReactNode => {
+      const maxRestarts = experiment.config.maxRestarts ?? 0;
+      return (
+        <>{record.restarts}{maxRestarts ? `/${maxRestarts}` : ''}</>
```

Comment:
@stoksc isn't max restarts always capped? can the user opt to make it go on forever?

Reply:
user opt makes it (0, max_int)